src/config/AppConstant.js

```diff
@@ -2,11 +2,15 @@ const API_BASE_URL = import.meta.env.DEV ? "http://localhost:8000/" : "https://a
 const CREATE_ACCOUNT_URL = "signup/";
 const LOGIN_ACCOUNT_URL = "login/";
 const LOGOUT_ACCOUNT_URL = "logout";
+const COLLECTIONS_URL = "vault_collections/";
+const ITEM_URL = "vault_items/"
 
 
 export {
     API_BASE_URL,
     CREATE_ACCOUNT_URL,
     LOGIN_ACCOUNT_URL,
-    LOGOUT_ACCOUNT_URL
+    LOGOUT_ACCOUNT_URL,
+    COLLECTIONS_URL,
+    ITEM_URL
```

Comment:
Nit: I would export these URLs out alphabetically, but it is a personally preference so that I can find things more easily